analyzers/tests/SonarAnalyzer.CSharp.Styling.Test/TestCases/MemberVisibilityOrdering.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
+
+public class ValidOrder
+{
+    public int publicOrInternal1;
+    internal int publicOrInternal2;
+    public int publicOrInternal3;
+    internal int publicOrInternal4;
+
+    protected int protectedVariant1;
+    private protected int protectedVariant2;
+    protected internal int protectedVariant3;
+    protected int protectedVariant4;
+    private protected int protectedVariant5;
+    protected internal int protectedVariant6;
+
+    private int private1;
+    private int private2;
+}
+
+public class AbovePrivate
+{
+    private int private1;
+
+    protected int protectedVariant1;            // Noncompliant {{Move this protected Field above the private ones.}}
+    private protected int protectedVariant2;    // Noncompliant {{Move this protected Field above the private ones.}}
+    protected internal int protectedVariant3;   // Noncompliant {{Move this protected Field above the private ones.}}
+
+    public int publicOrInternal1;               // Noncompliant {{Move this public Field above the private ones.}}
+    internal int publicOrInternal2;             // Noncompliant {{Move this internal Field above the private ones.}}
+    public int publicOrInternal3;               // Noncompliant {{Move this public Field above the private ones.}}
+    internal int publicOrInternal4;             // Noncompliant {{Move this internal Field above the private ones.}}
```

Comment:
I was confused by why all the issues had the same message before I realized they were all talking about the first field.

If not too much effort, we could add a secondary location.

Reply:
[#9174](https://github.com/SonarSource/sonar-dotnet/issues/9174)